src/test/java/com/wikia/webdriver/common/contentpatterns/AdsContent.java

```diff
@@ -72,13 +83,7 @@ public static String getSlotTrigger(String slotName) {
     SLOTS_SELECTORS = new HashMap<>();
     SLOTS_SELECTORS.put(BOTTOM_LB, "#BOTTOM_LEADERBOARD");
     SLOTS_SELECTORS.put(MOBILE_BOTTOM_LB, "#MOBILE_BOTTOM_LEADERBOARD");
-    SLOTS_SELECTORS.put(HOME_TOP_LB, "#HOME_TOP_LEADERBOARD");
-    SLOTS_SELECTORS.put(CORP_TOP_LB, "#CORP_TOP_LEADERBOARD");
-    SLOTS_SELECTORS.put(HUB_LB, "#HUB_TOP_LEADERBOARD");
-    SLOTS_SELECTORS.put(HUB_LB_GPT, "[id*='gpt/HUB_TOP_LEADERBOARD']");
     SLOTS_SELECTORS.put(TOP_LB, "#TOP_LEADERBOARD");
-    SLOTS_SELECTORS.put(HOME_MEDREC, "#HOME_TOP_RIGHT_BOXAD");
-    SLOTS_SELECTORS.put(CORP_MEDREC, "#CORP_TOP_RIGHT_BOXAD");
     SLOTS_SELECTORS.put(MEDREC, "#TOP_RIGHT_BOXAD");
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Define a constant instead of duplicating this literal "#TOP_RIGHT_BOXAD" 3 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AS1192)